doc/OnlineDocs/contributed_packages/pyros.rst

```diff
@@ -397,22 +397,23 @@ If we choose to designate all variables as either design or state variables, wit
   ...                                  options = {
   ...                                     "objective_focus": pyros.ObjectiveType.worst_case,
   ...                                     "solve_master_globally": True
-  ...                                   })
+  ...                                   }) # doctest: +ELLIPSIS
```

Comment:
Why is this adding `+ELLIPSIS`?

Reply:
That was supposed to be removed, I will remove it.